contracts/deploy/094_add_multisig_to_timelock.js

```diff
@@ -0,0 +1,56 @@
+const addresses = require("../utils/addresses");
+const { deploymentWithGovernanceProposal } = require("../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "094_add_multisig_to_timelock",
+    forceDeploy: false,
+    // forceSkip: true,
+    // onlyOnFork: true, // this is only executed in forked environment
+    reduceQueueTime: true, // just to solve the issue of later active proposals failing
+    proposalId: "",
+  },
+  async ({ ethers }) => {
+    const TIMELOCK_ADMIN_ROLE =
```

Comment:
Oddly enough we should not set the TIMELOCK_ADMIN_ROLE.

This permission is currently only given to the timelock contract itself, which allows for proposals to change the users, permissions, and timelock settings, but does not allow them to be called instantly, bypassing the timelock. Otherwise the multisig could instantly change the timelock to a short duration, then act using that short duration.

Reply:
That's a good catch, will get rid of it

Reply:
This is done as well